velox/exec/TableScan.cpp

```diff
@@ -251,7 +251,7 @@ RowVectorPtr TableScan::getOutput() {
       curStatus_ = "getOutput: updating stats_.rawInput";
       lockedStats->rawInputPositions = dataSource_->getCompletedRows();
       lockedStats->rawInputBytes = dataSource_->getCompletedBytes();
-      RowVectorPtr data = dataOptional.value();
+      RowVectorPtr data = std::move(dataOptional).value();
```

Comment:
Why do we need std::move here?

Reply:
```
using RowVectorPtr = std::shared_ptr<RowVector>;
```

And `RowVectorPtr data = dataOptional.value()` introduced a shared_ptr copying here

Reply:
I see. Could we just use` const auto& data = ...` here?

Reply:
Ah, we can also do that, don't know which one do we prefer. Basically, `RowVectorPtr data = std::move(...)` might also benifit from NRVO? because `return` might introduce a round of copying?

Reply:
Got it. Let's keep std::move. Thanks.